docs/ape17.rst

```diff
@@ -0,0 +1,562 @@
+   # astropy_helpers/MANIFEST.in, but requires additional includes for the actual
+   # package directory and egg-info.
+
+   include astropy_helpers/README.rst
+   include astropy_helpers/CHANGES.rst
+   include astropy_helpers/LICENSE.rst
+   recursive-include astropy_helpers/licenses *
+
+   include astropy_helpers/ez_setup.py
+   include astropy_helpers/ah_bootstrap.py
+
+   recursive-include astropy_helpers/astropy_helpers *.py *.pyx *.c *.h
+   recursive-include astropy_helpers/astropy_helpers.egg-info *
+   # include the sphinx stuff with "*" because there are css/html/rst/etc.
+   recursive-include astropy_helpers/astropy_helpers/sphinx *
+
+   prune astropy_helpers/build
+   prune astropy_helpers/astropy_helpers/tests
+
+Then add a new line near the top with the following::
+
+   include pyproject.toml
+
+Step 11 - Updating your documentation configuration
+---------------------------------------------------
+
+You will need to edit the ``docs/conf.py`` file to make sure it does not
+use astropy-helpers. If you see a code block such as:
+
+.. code:: python
+
+    try:
+       import astropy_helpers
+    except ImportError:
+       # Building from inside the docs/ directory?
+       if os.path.basename(os.getcwd()) == 'docs':
+           a_h_path = os.path.abspath(os.path.join('..', 'astropy_helpers'))
+           if os.path.isdir(a_h_path):
+               sys.path.insert(1, a_h_path)
+
+   # Load all of the global Astropy configuration
+   from astropy_helpers.sphinx.conf import *
+
+   # Get configuration information from setup.cfg
+   try:
+       from ConfigParser import ConfigParser
+   except ImportError:
+       from configparser import ConfigParser
+
+you should change it to:
+
+.. code:: python
+
+   try:
+       from sphinx_astropy.conf.v1 import *  # noqa
+   except ImportError:
+       print('ERROR: the documentation requires the sphinx-astropy package to be installed')
+       sys.exit(1)
+
+   # Get configuration information from setup.cfg
+   from configparser import ConfigParser
+   conf = ConfigParser()
+
+Find and replace any instances of ``package_name`` in the file with
+``name``.
+
+Step 12 - Setting up tox
+------------------------
+
+`tox <https://tox.readthedocs.io/en/latest/>`__ is a tool for automating
+commands, which is well suited to, e.g., running tests for your package or
+building the documentation. One of the benefits of using tox is that it
+will (by default) create a source distribution for your package and
+install it into a virtual environment before running tests or building
+docs, which means that it will be a good test of whether, e.g., you have
+declared the package data correctly.
+
+As a starting point, copy over the ``tox.ini`` file generated in :ref:`Step 0 <step-rerender>`.
+You can always then customize it if needed (although it should work out
+of the box).
+
+Once you have done this you should be able to do the following:
+
+Run tests with minimal dependencies::
+
+   tox -e test
```

Comment:
Trying this it seems to automatically run coverage as well; not sure if that's intended.

Reply:
I had deliberately just enabled it by default - for astropy core the performance penalty is large, but for most other packages I thought it might make sense to just always do it. But happy to also go with a set-up similar to in the core package if people would prefer (where coverage is opt-in with ``-cov``).

Reply:
Indeed, for my project, it is not a big deal, but I think then you also need to adjust the `tox.ini` to remove the `-cov` option. So it may be simpler to just make it the same. Which for me is also better - ideally everything is essentially the same as for astropy.

Reply:
I've now made it consistent with astropy core and made the ``cov`` option work